recirq/dfl/dfl_2d_second_order_trotter.py

```diff
@@ -7,10 +26,24 @@
 import numpy.typing as npt
 from tqdm import tqdm
 
+class TwoQubitGate2D(enum.Enum):
+    """Available two-qubit gate types for the Trotter simulation."""
+    CZ_SIMULTANEOUS = 'cz_simultaneous'
+    CPHASE_SIMULTANEOUS = 'cphase_simultaneous'
 
-class LGTDFL:
-    """
-    A class for simulating Disorder-free localization (DFL) on a 2-dimensional Lattice Gauge
+class InitialState(enum.Enum):
```

Comment:
Can we store these enums in a separate file since they can be shared by 1d and 2d files?